components/browser/menu/src/test/java/mozilla/components/browser/menu/item/BrowserMenuCaptionTest.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package mozilla.components.browser.menu.item
+
+import mozilla.components.browser.menu.R
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class BrowserMenuCaptionTest {
+
+    @Test
+    fun `browser menu caption uses correct layout`() {
+        val item = BrowserMenuCaption("Powered by Mozilla")
+        assertEquals(R.layout.mozac_browser_menu_item_caption, item.getLayoutResource())
+    }
+}
```

Comment:
Maybe a test to check the text is actually set? :)

Reply:
Ah, I didn't think there was a way to test this, but I looked at `BrowserMenuImageTextTest` for some inspiration. I've added a few more, but please let me know if that's not sufficient!